projects/storefrontlib/cms-components/navigation/search-box/search-box.component.html

```diff
@@ -127,6 +128,70 @@
       <cx-icon [type]="iconTypes.SEARCH"></cx-icon>
     </button>
   </label>
+  <label
+    *cxFeature="'a11yActiveAndFocusedSearchBoxComponent'"
+    class="searchbox cxFeat_a11ySearchboxLabel"
+    [class.dirty]="!!searchInput?.value"
+  >
+    <span class="cx-input-label">{{ 'common.search' | cxTranslate }}</span>
+    <div
+      class="cx-label-inner-container"
+      [cxFocus]="{ disableMouseFocus: true }"
```

Comment:
I think it we shouldn't need to use the VisibleFocusDirective here since it's already applied to the header.

Reply:
Updated it using` focus-within` pseudo instead.